### ISSUE TITLE
load the trained checkpoint error

### ISSUE THREAD

**Issue by zmlmanly** [david-gpu/srez#26]
Hi, I have a error when loading the trained checkpoint, and the result of trainning is same with your result. The code of loading checkpoint is
saver = tf.train.Saver()
saver.restore(sess, checkpoint_dir)
the error is 
Data loss:Invalid size in boundle entry:key global_step; stored size 8; expected size 4
Please help me!
Thank you very much!
